Formula/skaa.rb

```diff
@@ -0,0 +1,41 @@
+class Skaa < Formula
+  desc "Seven Kingdoms: Ancient Adversaries"
+  homepage "https://7kfans.com"
+  url "https://github.com/the3dfxdude/7kaa/releases/download/v2.15.5/7kaa-2.15.5.tar.xz"
+  sha256 "350a2681985feb4b71d20677d1a6d11921b9e3d97facbc94e4f020e848d8ab2b"
+  license "GPL-2.0-only"
+
+  # ARM is unsupported
+  depends_on arch: :x86_64
+
+  depends_on "enet"
+  depends_on "gettext"
+  depends_on "sdl2"
+
+  uses_from_macos "curl"
+
+  on_macos do
+    depends_on "gcc" => :build
+  end
+
+  fails_with "clang"
+
+  def install
+    # ensure we use gcc for building the app, Clang will fail
+    ENV["CXX"] = "/usr/local/bin/g++-12"
+
+    system "./configure", "--prefix=#{prefix}"
```

Comment:
```suggestion
  fails_with :clang

  def install
    system "./configure", *std_configure_args
```
Please don't hardcode a path to the compiler.

Reply:
The problem here is, if I don't hardcode the path, it seems the Clang front-end for `gcc` is used.

Reply:
It won't.